libs/sdk-js/src/client.ts

```diff
@@ -983,6 +986,137 @@ export class RunsClient extends BaseClient {
   }
 }
 
+export class StoreClient extends BaseClient {
+  /**
+   * Store or update an item.
+   *
+   * @param namespace A list of strings representing the namespace path.
+   * @param key The unique identifier for the item within the namespace.
+   * @param value A dictionary containing the item's data.
+   * @returns Promise<void>
+   */
+  async putItem(
+    namespace: string[],
+    key: string,
+    value: Record<string, any>,
+  ): Promise<void> {
+    namespace.forEach((label) => {
+      if (label.includes(".")) {
+        throw new Error(
+          `Invalid namespace label '${label}'. Namespace labels cannot contain periods ('.')`,
+        );
+      }
+    });
+
+    const payload = {
+      namespace,
+      key,
+      value,
+    };
+
+    return this.fetch<void>("/store/items", {
+      method: "PUT",
+      json: payload,
+    });
+  }
+
+  /**
+   * Retrieve a single item.
+   *
+   * @param namespace A list of strings representing the namespace path.
+   * @param key The unique identifier for the item.
+   * @returns Promise<Item>
+   */
+  async getItem(namespace: string[], key: string): Promise<Item> {
+    namespace.forEach((label) => {
+      if (label.includes(".")) {
+        throw new Error(
+          `Invalid namespace label '${label}'. Namespace labels cannot contain periods ('.')`,
+        );
+      }
+    });
+
+    return this.fetch<Item>("/store/items", {
+      params: { namespace: namespace.join("."), key },
+    });
```

Comment:
Here:
```suggestion
    const getRes = await this.fetch<Item>("/store/items", {
      params: { namespace: namespace.join("."), key },
    });
    
    if (!getRes.ok) {
      throw new Error("...")
    }
    return {
      ...getRes,
      createdAt: getRes.created_at,
      updatedAt: getRes.updated_at,
    }
```

Not exactly sure if this typing is all correct, but roughly this idea

Reply:
ack - thanks!